src/routes/+page.svelte

```diff
@@ -8,6 +16,10 @@
 <h1>Welcome to SvelteKit</h1>
 <p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
 
+<button class="button" on:click={viewSomeCampaign}>
```

Comment:
button html elements require `type` for accessibility reasons.

Reply:
I typed in a type for my button-type button